src/Main.js

```diff
@@ -1,12 +1,35 @@
 import { Component } from 'react';
 import HornedBeast from './HornedBeast'
 
+
 class Main extends Component {
   render() {
+    data.map(beast => { 
+     
+
+    } )
```

Comment:
Correct so far, nice job!

To help a bit, so you started to map over the data, and inside this map, you want to render a bunch of HornedBeast components. 

We also know that the HornedBeast component should have props passed of `title`, `image`, and `description`. 
Inside the map, you will use your return statement as usual and the open `(`. Then add your <HornedBeast />` component with all the props: `title={beast.title}` etc. 👍